tests/chainer_tests/functions_tests/pooling_tests/test_upsampling_2d.py

```diff
@@ -25,12 +25,10 @@ def setUp(self):
             -1, 1, self.in_shape).astype(self.dtype)
         self.ggx = numpy.random.uniform(
             -1, 1, self.pooled_y.shape).astype(self.dtype)
+        self.check_backward_options = {}
```

Comment:
Should we declare `self.check_backward_options` although it seems to remain an empty dict throughout the unit test?

Reply:
It can be removed, as you say, but it can be also left because it is not harmful and we could just put tolerance when we want to adjust it in the future. Anyway, it doesn't have very high priority, I think.

Reply:
I agree that it's a minor concern.